lib/event_tracer/dynamo_db_log_worker.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
```

Comment:
Could we add a check here, if `sidekiq` cannot be loaded then we raise error saying user should add `sidekiq` to Gemfile?